mmtrack/models/track_heads/quasi_dense_embed_head.py

```diff
@@ -0,0 +1,133 @@
+import numpy as np
+import torch
+import torch.nn as nn
+from mmdet.models import HEADS, build_loss
+
+from mmtrack.core import embed_similarity
+from .roi_embed_head import RoIEmbedHead
+
+
+@HEADS.register_module()
+class QuasiDenseEmbedHead(RoIEmbedHead):
+
+    def __init__(self,
+                 embed_channels=256,
+                 softmax_temp=-1,
+                 loss_track=dict(
+                     type='MultiPosCrossEntropyLoss', loss_weight=0.25),
+                 loss_track_aux=dict(
+                     type='L2Loss',
+                     sample_ratio=3,
+                     margin=0.3,
+                     loss_weight=1.0,
+                     hard_mining=True),
+                 *args,
+                 **kwargs):
+        super(QuasiDenseEmbedHead, self).__init__(*args, **kwargs)
+        self.relu = nn.ReLU(inplace=True)
+        self.convs, self.fcs, last_layer_dim = self._add_conv_fc_branch(
```

Comment:
The line can be deleted
Please delete the duplicated codes that are already in `super(QuasiDenseEmbedHead, self).__init__(*args, **kwargs)`